Arithmetic/AverageSum.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace CSharp_Practice.Arithmetic
+{
+    using Base;
+
+    public class AverageSum
+    {
+        public static void Average()
+        {
+            PrintInstructions.PrintLine("Input 4 numbers and it will take the average of the 4 numbers.");
+            int num1 = InputChecker.InputInteger();
+            int num2 = InputChecker.InputInteger();
+            int num3 = InputChecker.InputInteger();
+            int num4 = InputChecker.InputInteger();
+
+            float average = (num1 + num2 + num3 + num4) / 4;
```

Comment:
Double check that you ever end up with non-integer results.  Since num1-4 are ints, C# probably assumes 4 also is, and does the division operator on the integers.  You can get around this simply by ensuring one of the division operands is a float, for example by writing it as 4.0f instead of 4